devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,415 @@
+and why we decided to participate in it.
+
+We will share our experiences and lessons learned.
+
+Finally, we will think aloud why this program may or may not be
+for you.
+
+Why we do this?
+---------------
+
+* To help breed software engineers
+* To sustain the open source culture
+* Promote diversity in the profession
+
+.. Things to talk about ^ (ietingof)
+
+  Much like in medieval European trade guilds (or even nowadays science),
+  software engineering seems to require some kind of apprenticeship before
+  one can attain professional proficiency.
+
+  As creating and sustaining open source software is part of the Red Hat
+  business strategy, attracting and growing open source engineers lines-up
+  well with the company's goals.
+  
+.. Things to talk about ^ (rpioso)
+
+  Promoting diversity is a core part of OpenStack, Dell EMC, and Red Hats'
+  cultures. Not only is that the right thing to do, diverse teams produce
+  superior software and business results.
+
+.. Things to talk about ^ (ietingof)
+
+  Beyond that, we are open source engineers ourselves. It is in our direct
+  interest to sustain and promote the culture we enjoy being a part of.
+
+What is OpenStack?
+------------------
+
+* Cloud management software stack and infrastruture software
+* Decade-old open source community project
+* Has involved 1025 developers from 126 companies
+* Made out of 728 repos
+
+Reference: https://docs.openstack.org/project-team-guide/introduction.html
+Reference: https://www.stackalytics.com, Stein cycle
+
+.. Things to talk about ^ (rpioso)
+
+  OpenStack is a large, well-established, and highly collaborative open source
+  software engineering project. At OpenStack, we crave quality engineering
+  practices, mutual respect, and continuous learning.
+
+  The goal of the OpenStack project is to create and maintain free and open
+  source cloud management and infrastructure software.
+  
+  The first reference offers a bit of OpenStack history.
+
+  If we look at Stacklytics for the current release cycle (Stein), the basic
+  stats can be seen on the slide.
+
+About Outreachy
+---------------
+
+What is Outreachy?
+
+  * Volunteer organization under Software Freedom Conservancy
+  * Brings together interns, mentors, and code
+  * Promotes diversity and free software
+
+How does it work?
+
+  * Interns get on board open source projects
+  * Each intern gets a mentor (or two)
+  * 3 month paid internship
+
+.. Things to talk about ^ (ietingof)
+
+  Outreachy is a volunteer organization operating under the Software Freedom
+  Conservancy umbrella.
+
+  Outreachy helps bring together interns, mentors, and open source projects
+  that seek contributors and mentors.
+
+  The main focus of Outreachy is to promote and ensure diversity, getting
+  different people from all over the world to join the open source movement.
+
+Why become an intern?
+---------------------
+
+* To find your aspirations
+* To learn the software engineering craft
```

Comment:
Change "software engineering craft" to "software engineering, documentation, design, or other craft"

Reply:
Done